samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication/app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
```

Comment:
`pnp`? Just curious.

Reply:
No idea. I think that came over from the enterprise sample

Reply:
I think it has to do with the Fluent Dependency